00-programming-fundamentals/01-how-to-osx.md

```diff
@@ -1,55 +1,150 @@
+Delete  | Delete :p
+Page up & Page Down | Moves the window up or down but does **not** move the cursor. 
+Home/End | Jumps to the bottom of the document but does **not** move the cursor. 
+Windows + Home/End | Jumps **with** cursor to top/bottom of document
+Numlock | Nothing, nada
+10-key-pad | Also nothing
+
+
+
+
+
+## Taking Screenshots
+
+Screenshots on a Mac are very cool. By default, they save a file to the desktop. If you add Control (⌃) to the shortcut, the screenshot will be copied to the clipboard instead.
+
+
+Behavior / PC Key | Mac Keyboard Shortcut | In Symbols!
+------------------|-----------------------|------------
+Print Screen (full screen, saved to desktop) | Command + Shift + 3 | ⌘⇧3 (⌘#)
+Print Screen (full screen, copied to clipboard) | Control + Command + Shift + 3 | ⌃⌘⇧3 (⌃⌘#)
+Print Screen (select area, saved to desktop) | Command + Shift + 4 | ⌘⇧4 (⌘$)
+Print Screen (select area, copied to clipboard) | Control + Command + Shift + 4 | ⌃⌘⇧4 (⌃⌘$)
+Print Screen (application window, PC: Alt + Print Screen) | Command + Shift + 4, Space-bar, Click on Window | ⌘⇧4 (⌘$), {Space}, Click on Window
+
+
+<a id="#custom-shortcuts"></a>
+
+## Custom Shortcuts
+
+
+There might be applications that will do some or all of this for you, but I opted to make the changes manually. Some function keys were easier to reprogram than others. You'll need to open __System Preferences__ and go into __Keyboard__ settings. In the __Shortcuts__ tab, check out your options in the list on the lift. It should say things like __Launchpad & Dock__, __Mission Control__, and __Keyboard__. Click on __App Shortcuts__ at the bottom. Now you can customize your own shortcuts!
+
+To create a shortcut, click the __+__ below the right options list. You'll see right away that you can choose between creating a shortcut for all applications or just for individual ones. The next thing you'll see is a field that asks for the __Menu Title__. To create a shortcut, all you have to do is find the menu option and type _exactly_ what it says into this field. Finally, click or tab over to the __Keyboard Shortcut__ field, and press the shortcut you'd like this functionality to be associated with. Hit the __Add__ button, and you're finished. It should work now. Try out your new keyboard shortcut and make sure it works right!
+
+Unfortunately, not all applications use the same menu options for the same functions, so sometimes you may have to create multiple shortcuts to add the functionality you are looking for. For example, to restore the _F11_ / _Full Screen_ functionality to my Macbook, I had to create four separate entries in the Shortcuts creator. For restore the _F1_ / _Help_ dialog functionality, I had to create a separate shortcut for every application. So keep this in mind! Some things are easier to create as shortcuts than others.
+
+---
+## Resources
+*  [Jeri's Guide to Going from PC to Mac](https://gist.github.com/drvonnjerryxlii/9898254abef2d184c659)
```

Comment:
I think it's important to explicitly call out that we pulled a bunch/some of this stuff from this guide originally